test/geocoder-test.js

```diff
@@ -125,7 +161,30 @@ describe('Testing geocoder', function() {
     geocoder.geocodeAddress(mockAddress)
       .catch(error => {
         should(error).be.an.Error;
-        should(error.message).equal('Wrong clientId or privateKey');
+        should(error.message).equal('Authentication error');
+      })
+      .then(done, done);
+  });
+
+  it('should throw authentication error when using api key', function(done) {
```

Comment:
I guess normally it shouldn’t throw an error. Specify under what conditions it throws an error.